server/src/test/java/keywhiz/service/daos/SecretSeriesMapperTest.java

```diff
@@ -17,9 +19,17 @@
 
 @RunWith(KeywhizTestRunner.class)
 public class SecretSeriesMapperTest {
-  @Inject private SecretSeriesMapper mapper;
+  @Inject private SecretSeriesMapper.SecretSeriesMapperFactory mapperFactory;
+  @Inject private DSLContext jooq;
   @Inject @Readwrite private GroupDAO groupDAO;
 
+  private SecretSeriesMapper mapper;
+
+  @Before
+  public void before() {
+    mapper = mapperFactory.using(jooq);
+  }
```

Comment:
Since we can no longer directly inject the SecretSeriesMapper, we need a bit more setup code to keep our test working.